clockwork_test.go

```diff
@@ -87,55 +89,6 @@ func TestFakeClockAfter(t *testing.T) {
 	}
 }
 
-func TestNotifyBlockers(t *testing.T) {
```

Comment:
I don't think I understand what you mean.

This underlying function `notifyBlockers`, was merged into `setExpirer`. My reasoning was:
1. `notifyBlockers` was a for loop that, well, notified blockers, but this was only done by `setExpirer`. Since `setExpirer` doesn't have much logic in it, it seemed reasonable to add the functionality there. I did add a comment, thinking maybe that helps?
2. `notifyBlockers` was a private function, as is `setExpirer`. It is tested via the public methods for `Timers`, `Tickers`, and `BlockUntil*`. I've always followed the "test public functions" mantra, which I think is intended to ensure we test behavior of the public API rather than implementation details.

Reply:
Apologies, I mixed `notifyBlockers` up with `BlockUntil`. For some reason, I thought this was supposed to test the "deprecated" method.